services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpointImpl.java

```diff
@@ -36,23 +47,112 @@
     }
 
     @Override
-    public RegisteredApplication registerApplication(RegisteredApplication application)
+    public RegisteredApplication getApplication(int id) {
```

Comment:
Please setup your IDE to use the code style specified here: https://github.com/windup/windup/

Unless everyone wants to use a different style for windup-web?